pallets/parachain-staking/src/lib.rs

```diff
@@ -2097,6 +2150,75 @@ pub mod pallet {
 			}
 		}
 
+		/// Annually reduce the reward rates for collators and delegators.
+		///
+		/// # <weight>
+		/// Weight: O(1)
+		/// - Reads: LastRewardReduction, InflationConfig
+		/// - Writes: LastRewardReduction, InflationConfig
+		/// # </weight>
+		fn adjust_reward_rates(now: T::BlockNumber) -> Weight {
+			let year = now / T::BlocksPerYear::get();
```

Comment:
Although unlikely, nothing prevents `BlocksPerYear` to be equal to 0.

Reply:
T::BlocksPerYear::get().max(1)